pyaerocom/griddeddata.py

```diff
@@ -2532,10 +2532,13 @@ def std(self):
 
     def _check_lonlat_bounds(self):
         """Check if longitude and latitude bounds are set and if not, guess"""
-        if self.longitude.bounds is None:
+        if not self.longitude.has_bounds():
             self.longitude.guess_bounds()
-        if self.latitude.bounds is None:
-            self.latitude.guess_bounds()
+        if not self.latitude.has_bounds():
+            try:  # needs at least 2 values to work
+                self.latitude.guess_bounds()
+            except ValueError:
+                pass
```

Comment:
Could a test be written for this?

Reply:
According to the comment, it needs 2 values to work.
Why not to write it this?
``` python
            if self.latitude.size > 2
                self.latitude.guess_bounds()
```

Reply:
@avaldebe `self.latitude` is of type `DimCoord` which does not have the size property, the try statement is the shortest way to test (but probably not the most pyhtonic). May I leave it as it is?

Reply:
@lewisblake a test has been added